tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -43,6 +43,44 @@
   register: edpm_privatekey
   when: edpm_encoded_privatekey is undefined
 
+# Remove this when https://code.engineering.redhat.com/gerrit/q/topic:stable-compute-uuid
+# is released in 17.x
+- name: Temporary fix to ensure stable compute UUID
+  block:
+    - name: ensure SSH key
+      ansible.builtin.copy:
+        dest: /tmp/ansible_private_key
+        content: "{{ edpm_encoded_privatekey | default(edpm_privatekey.content) | b64decode }}"
+        mode: "0600"
+      when: edpm_privatekey_path is undefined
+
+    - name: populate compute_id file
+      ansible.builtin.shell: |
+        {{ shell_header }}
+        {{ oc_header }}
+        declare -A computes
+        computes=(
+          ["{{ edpm_node_hostname }}.localdomain"]="{{ edpm_node_ip }}"
+        )
+
+        for name in "${!computes[@]}";
+        do
+          uuid=$(\
+            oc exec -t openstackclient -- \
+              openstack compute service list \
+              --service nova-compute \
+              --host $name \
+              -f value -c 'ID'\
```

Comment:
OK. My bad. This is the compute service UUID but we need the compute node UUID instead.

Reply:
switched to `openstack hypervisor show`